types/interfaces.ts

```diff
@@ -20,18 +20,13 @@ export interface ServicePost {
   slug: string
 }
 
-export interface EmployeePost {
```

Comment:
Kindly add the removed interface again. and keep both. i.e. the new one and old one